simulation/behavior_tree_plugin/src/action_node.cpp

```diff
@@ -505,30 +506,73 @@ auto ActionNode::calculateUpdatedEntityStatusInWorldFrame(
   -> traffic_simulator::EntityStatus
 {
   using math::geometry::operator*;
+  using math::geometry::operator+;
+  using math::geometry::operator+=;
+
+  constexpr bool desired_velocity_is_global{false};
+
+  const auto include_crosswalk = [](const auto & entity_type) {
+    return (traffic_simulator_msgs::msg::EntityType::PEDESTRIAN == entity_type.type) ||
+           (traffic_simulator_msgs::msg::EntityType::MISC_OBJECT == entity_type.type);
+  }(canonicalized_entity_status->getType());
+
+  const auto matching_distance = default_matching_distance_for_lanelet_pose_calculation;
+
+  const auto buildUpdatedPose =
```

Comment:
We use snake_case for function object in this codebase.
```suggestion
  const auto build_updated_pose =
```

Reply:
Ok, it's done – now the function object name now uses snake_case.